apps/next-evans/__tests__/seo.spec.ts

```diff
@@ -0,0 +1,99 @@
+import { describe, it, expect } from 'vitest';
+import { documentPageDescription, friendPageDescription } from '../lib/seo';
+
+describe(`friendPageDescription()`, () => {
```

Comment:
these tests are nice 👍

...but did you check on and convert my old tests for this? the file is called `seo.spec.ts`

speaking of which, maybe take a look at https://vitest.dev/api/#test-each and make sure you're clear on the `test.each` style of tests, i find i use it probably 50% or more of the time, it allows you to write more tests with less boilerplate, and perhaps most importantly, decreases my reluctance to add another test.

you don't necessarily have to delete these tests, but ideally we should keep the others at least as well, there may be hard-won edge cases covered there 👍